manifests/init.pp

```diff
@@ -57,6 +58,7 @@
   Array $environment                 = [],
   String $package_name               = 'certbot',
   $package_ensure                    = 'installed',
+  String[1] $dnfmodule_version           = 'python36',
```

Comment:
You can't set this to `python36` since it's only valid for EL8 but it's invalid for anything else. It would be best to create an 8.yaml [here](https://github.com/voxpupuli/puppet-letsencrypt/tree/master/data/os/RedHat) with `python36` and then set it to `~` in [common.yaml](https://github.com/voxpupuli/puppet-letsencrypt/blob/master/data/common.yaml). Then change it to
```suggestion
  Optional[String[1]] $dnfmodule_version,
```